app/models/User.js

```diff
@@ -114,6 +114,7 @@ const UserSchema = new mongoose.Schema({
       },
       education: [
         {
```

Comment:
all backend pr need to be with atleast 2  test cases 1 pass 1 fail

Reply:
can you please be more specific , what you mean?